packages/contentstack-import/src/import/modules-js/workflows.js

```diff
@@ -147,6 +147,15 @@ module.exports = class importWorkflows {
         })
         .catch(function (error) {
           fileHelper.writeFileSync(workflowFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).workflow !== undefined) {
+            if (JSON.parse(error.request.data).workflow.name) {
+              message_content_type =
+                ' Failed workflow: ' + JSON.parse(error.request.data).workflow.name;
+            }
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code